src/model/spec/userProfile.spec.ts

```diff
@@ -0,0 +1,33 @@
+import { expect } from 'chai';
+import * as deepFreeze from 'deep-freeze';
+import { UserProfile } from '../userProfile'
+import {} from 'mocha'
+import {} from 'core-js'
+
+describe('userProfile', () => {
+  let userProfile = null;
+
+  beforeEach(() => {
+    userProfile = new UserProfile();
+  })
```

Comment:
No prob :), anyway it has been a good review, the beforeeach thing it's a good práctice. My plan is to merge this tomorrow and try to make an integration with Travis live in the class session (for sure it Will Be a disaster :)). Later on we can open a branch to enhance or change pieces on the testing infraestructura side, I'm keen on reviewing what you propose, karma sometimes is kind of pain in the neck.